warehouse/query-core/src/test/java/datawave/query/util/WiseGuysIngest.java

```diff
@@ -791,6 +791,24 @@ public static void writeItAll(Connector con, WhatKindaRange range) throws Except
             mutation.put(ColumnFamilyConstants.COLF_TF, new Text(datatype), emptyValue);
             bw.addMutation(mutation);
             
+            // for testing #NOEXPANSION function
```

Comment:
I must be missing something.  I don't see the test where you actually pass this param.

Reply:
The commit with the test got lost in a rebase, I'll re-add it shortly.